src/profile/scripts/directives.js

```diff
@@ -24,7 +24,9 @@
                             scope.fields = response.data.actions.PUT;
                         });
 
-                    scope.editProfile = function () {
+                    scope.editProfile = function (localStorageVar) {
```

Comment:
I don't think other HTTP objects have an appropriate place to use this functionality, as it's only applicable when changing the profile.